packages/commcare/src/Adaptor.js

```diff
@@ -82,6 +82,51 @@ export function get(path, params = {}, callback = s => s) {
   };
 }
 
+/**
+ * Make a post request to commcare
+ * @example
+ * post(
+ *   "user",
+ *  {"username":"test",
+ *  "password":"somepassword"},
+ *   (state) =< state.data
```

Comment:
Little typo on the arrow function

I would actually remove this though, because it complicates the example and for no particular benefit